worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -402,6 +404,11 @@ private void trackProgress(ProgressTracker progressTracker) {
                 .collect(Collectors.toList());
         jobOutputs.addAll(errorJobOutputs);
 
+        if (jobOutputs.isEmpty()) {
+            var errMsg = "No JobOutput records were created.";
```

Comment:
Will this message propagate all the way to the end user? If yes, let's make it somewhat less technical, i.e. "The export process has produced no results".

Reply:
done - error message changed.